src/undomanager.js

```diff
@@ -7,7 +7,7 @@ define([
     $,
     _,
     util,
-    UNDO_ALERT
+    undo_alert
```

Comment:
aren't these usually uppercase by vellum convention? Or is that just for test xml?